kivymd/uix/label.py

```diff
@@ -311,6 +311,7 @@ def _set_text(self, value):
 
     def __init__(self, **kwargs):
         super().__init__(**kwargs)
+        self.check_font_styles()
```

Comment:
This line will only be executed in Python code, and be avoided completely in KVlang. You should move it to a call after `__init__` is done.

Reply:
Oops I missed that out. Thanks for raising this up will use the `Clock.schedule_once` to schedule the check_function